duckduckgo/BrowserServicesKit
# Enable unit tests via Xcode

Task/Issue URL: https://app.asana.com/0/1200194497630846/1201743270866809/f
Tech Design URL:
CC: @tomasstrba 

**Description**:

This PR updates BSK such that unit tests can run both via `swift test` and from within Xcode.

I'm not clear on why this differs between Xcode and `swift test`, but using the `.copy` rule preserves the directory structure of our resources and requires that path to be specified in the bundle. For example, leaving `.copy` in place means that we would need to prepend `Resources/` to the paths when looking them up during test runs.

Rather than do this, I've changed the rules to `.process` which turns the bundle into a flat structure.

I also removed a redundant test file, which we had two copies of under the same name but in two different directories.

**Steps to test this PR**:
1. Check out this branch
2. Run tests via `swift test`
3. Run tests via Xcode



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
